src/runtime.ts

```diff
@@ -88,9 +88,21 @@ export class BaseAPI {
             }
         }
         let response = await (this.configuration.fetchApi || fetch)(fetchParams.input, fetchParams.init);
-        let response_for_post_processing = response.clone()
+
+        /**
+         * Node-fetch package has a `highwatermark` setting which in some versions defaults to 16kb,
+         * which is far too low for many server contexts and causes response.clone() to freeze the app.
+         * For this reason, be very careful about using post-processing middlewares below node version
+         * 18 when you can use a fetch API built into the node server without that limitation.
+         */
+        let response_for_post_processing: Response | null = null;
```

Comment:
i think this looks fine to me - mb you couldve used a const and only run response.clone if there is a middleware with post processing but idk if that makes up for the 'let' vs 'const' difference